### PR TITLE
C2.0 fixes to dev

### PR DESCRIPTION
**Pull request by 1sourcecontrol**
Fixed an ambiguity in the classes doc.
Merged class starting money numbers.
Nerfed "Needs More Dakka" nanite cost to 15.
Altered Gunslinger's "Protective Awareness" Feat
Removed Mage and Sorcerer.
Buffed Monk so that it's a little more playable in C2.0 -- I think there is potential to have some kind of Zealot class, that at least has more purchasing power than the existing monk and actually fits our aesthetic. I didn't want to yank Monk because it's actually playable, incase anyone wants to, until we redo the classes.
Various text fixes.
Removed my dev note from "Rainman" and buffed rainman -- tell me what you think!
Clarified text in "Positional Mastermind" and gave it a slight nerf (to prevent abuse by gunslinger).
Modified "Will to Survive" to prevent purging death and whatnot.

### CODE REVIEW

File: CharacterCreation/03_Feats.txt
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt]: Propose "Any enemy" instead of "Anything" as we don't want to make the friendly medic's job harder.
Similarly "Living enemy" instead of "living creature".

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt]: Can we reword this snippet? While I don't care all that much about comas, a sentence should not be made out of four sentence fragments.

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:37]: Great clarification! Thank you!

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:102]: Good catch!

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:125]: Another good catch!

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:441]: Good yank

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [inline CharacterCreation/02_Classes.txt:392]: Is Opportunity Attack defined somewhere?

File: CharacterCreation/03_Feats.txt
Comment (bleehu) [inline CharacterCreation/03_Feats.txt:260]: I like it!

**Review by bleehu — CHANGES_REQUESTED** [review]
Some minor changes requested. Once these changes are made, you may shipit and shortcut the normally required second review.

**Review by Turtlelord26 — COMMENTED** [review]
Would request one noted change but no merge-blockers.

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [reply CharacterCreation/02_Classes.txt:392]: It is, actually. Only they are currently called "Penalty attacks." https://github.com/bleehu/Compound_X/blob/play/Basic%20Rules.txt#L427

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt]: #199. Well, it looks like you invalidated your own comment by making a separate pull request to change something i'm already changing?

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt:37]: I appreciate the compliment, but I changed how Battle Cry works, rather than clarified it. Are you talking about something else?

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt:392]: We should define it better in the glossary

File: CharacterCreation/03_Feats.txt
Comment (1sourcecontrol) [reply CharacterCreation/03_Feats.txt:260]: Thanks! :)

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt:37]: Ah you must be referring to the roar itself

File: CharacterCreation/03_Feats.txt
Comment (1sourcecontrol) [reply CharacterCreation/03_Feats.txt]: I had gone for implication instead of stating clearly. Toupee. Thanks!

File: CharacterCreation/02_Classes.txt
Comment (bleehu) [reply CharacterCreation/02_Classes.txt:37]: my bad. I was reading fast and thought [your own roll too] was added by you.

File: CharacterCreation/03_Feats.txt
Comment (bleehu) [reply CharacterCreation/03_Feats.txt]: donald-trump-toupee

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt:37]: no worries! :)

File: CharacterCreation/02_Classes.txt
Comment (1sourcecontrol) [reply CharacterCreation/02_Classes.txt:392]: BTW, do you think this is a good change? (I'm guessing you do, since you didn't say anything about it, but verification is nice)

**Pull request merged**
